### ISSUE TITLE
Inaccurate Intructions

### ISSUE THREAD

**Issue by DivineIntelLLC** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#10617]
[Enter feedback here]

The instructions do not match the menu options:

"You must first purchase a Phone System add-on license. To do that, [sign in to the Microsoft 365 admin center and choose Billing](https://go.microsoft.com/fwlink/p/?linkid=868433) > Purchase services > Add-on subscriptions > Buy now."
---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 33bfc660-573d-b57e-8b67-6aa1fbe820ad
* Version Independent ID: 53318fba-10e9-cd4b-f0ee-397cd10e8c0c
* Content: [Microsoft Calling Plans for Teams - Microsoft Teams](https://learn.microsoft.com/en-US/microsoftteams/calling-plans-for-office-365?WT.mc_id=TeamsAdminCenterCSH)
* Content Source: [Teams/calling-plans-for-office-365.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/public/Teams/calling-plans-for-office-365.md)
* Service: **msteams**
* GitHub Login: @CarolynRowe
* Microsoft Alias: **crowe**

**Comment by get-itips** [comment]
Hello @DivineIntelLLC how are you?
What are you seeing in your case?
Thanks

**Comment by DivineIntelLLC** [comment]
I provided those details in my original submission. The menu options from the instructions do not match the menu options in the current interface.

**Comment by get-itips** [comment]
Yes, however, I would like to understand the option that you are seeing from your point of view, in which step you can not proceed? Thanks
